rollup.config.js

```diff
@@ -0,0 +1,99 @@
+const resolve = require('rollup-plugin-node-resolve');
+const commonjs = require('rollup-plugin-commonjs');
+const replace = require('rollup-plugin-replace');
+const { uglify } = require('rollup-plugin-uglify');
+const serve = require('rollup-plugin-serve');
+const livereload = require('rollup-plugin-livereload');
+const license = require('rollup-plugin-license');
+const json = require('rollup-plugin-json');
+const { argv } = require('yargs');
+
+const pkg = require('./package.json');
+
+const isProduction = argv.prod === true;
+
+const getPlugins = isProduction => {
+  return [
+    resolve({
+      browser: true
+    }),
+    commonjs(),
+    json(),
+    replace({
+      __DEV__: isProduction ? 'false' : 'true',
```

Comment:
`! isProduction `

Reply:
Not sure what that means 😬

Reply:
Simpler statement, same meaning

Reply:
nope, I need the result to be a string, so it'd have to be either `(!isProduction).toString()` or what I have there